config_files/tum_odometry.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML:1.0
 camera_calibration_file: "/home/sarmento/rgbd_rtk/config_files/kinect_default.yaml"
-index_file: "/home/marcos/Downloads/TUM_Datasets/rgbd_dataset_freiburg1_xyz/index"
```

Comment:
Please, change this file as follows:
camera_calibration_file: "/path/to/camera_calibration.yaml"
index_file: "/path/to/dataset/dataset_xyz/index"

In this way, the dataset loader (e.g. RGBDLoader) will exit the program with an error message informing that the dataset doesn't exist, forcing the user to change its own local copy of this file.